src/html-conventions-lib/association-html-conventions.xsl

```diff
@@ -26,6 +26,55 @@
     <xsl:template match="connector[./properties/@ea_type = 'Association']">
         <xsl:variable name="associationChecks" as="item()*">
             <xsl:if test="f:checkIfConnectorTargetAndSourceElementsExists(.)">
+                <!--    Start of common connectors checkers rules     -->
+                <xsl:call-template name="co-namingFormat">
+                    <xsl:with-param name="connector" select="."/>
+                </xsl:call-template>
+                <xsl:call-template name="co-missingNamePrefix">
+                    <xsl:with-param name="connector" select="."/>
+                </xsl:call-template>
```

Comment:
I wonder if we can leverage the power of select=".".

We don't need to, but I see this as a convenient and powerful mechanism that you implemented. 
So lower level logic can be unified and the variations handled at this level.